src/me/EtienneDx/RealEstate/Transactions/ClaimRent.java

```diff
@@ -107,8 +122,8 @@ public boolean update()
 			else if(sign.getBlock().getState() instanceof Sign)
 			{
 				Sign s = (Sign) sign.getBlock().getState();
-				s.setLine(0, RealEstate.instance.config.cfgSignsHeader);
-				s.setLine(1, Utils.getSignString("Rented by " + Bukkit.getOfflinePlayer(buyer).getName()));
+				s.setLine(0, ChatColor.GOLD + "[Rented]"); //Changed the header to "[Rented]" so that it won't waste space on the next line and allow the name of the player to show underneath.

```

Comment:
Could you move this line to a config line, and maybe make the same for the other options (sell & lease)

Reply:
My pleasure. 

To answer your question, no the iml file was not really supposed to be committed it kinda was made by my IDE (Intellij) automatically and I did not realize it at first. I should have probably removed it but I did not put that much thought into it tbh. I'll try to do get rid of it in the next commit and add the new things you suggested. 

Though that might take a week or so, as I have other priorities at the moment.

Reply:
Alright thanks, I'll keep an eye on the PR :)